lib/compiler/src/core_pp.erl

```diff
@@ -72,6 +76,14 @@ maybe_anno(Node, Fun, #ctxt{clean=true}=Ctxt) ->
 	    end
     end.
 
+needs_line_anno(Node) ->
+    case cerl:is_c_primop(Node) andalso
+        cerl:concrete(cerl:primop_name(Node)) of
```

Comment:
Nitpick: without parentheses, Emacs-formatting places `cerl:concrete(...)` on the same line as the alternatives, which I find difficult to read when super-tired.
```suggestion
    case (cerl:is_c_primop(Node) andalso
          cerl:concrete(cerl:primop_name(Node))) of
```